cmd/blockchaincmd/import_public.go

```diff
@@ -191,6 +191,7 @@ func importBlockchain(
 	if err != nil {
 		return models.Sidecar{}, nil, err
 	}
+	printFunc("  Sovereign: %v", !subnetInfo.IsPermissioned)
```

Comment:
I dont think we need to print this. Let's just print if blockchain is Subnet (non sovereign)